packages/text-document-utils/test/range.test.ts

```diff
@@ -0,0 +1,177 @@
+import { Position, Range } from 'vscode-languageserver-types';
+import { arePositionsEqual, areRangesEqual, createRange, createRangeWithPosition, rangeAt } from '../src/range';
+
+describe('Range utility functions', () => {
+    test('arePositionsEqual', () => {
+        const result1 = arePositionsEqual(Position.create(0, 0), Position.create(0, 0));
+        const result2 = arePositionsEqual(Position.create(1, 0), Position.create(0, 0));
+        const result3 = arePositionsEqual(Position.create(0, 1), Position.create(0, 0));
+
+        expect(result1 && !result2 && !result3).toBe(true);
```

Comment:
Mixing multiple test cases in the same `expect` does not look right. Please split this into separate `test` blocks or use parametrised tests. Also test case names should be unique so they can be distinguished from each other.

Reply:
done.